src/crimson/os/seastore/cache.cc

```diff
@@ -1189,6 +1189,7 @@ void Cache::complete_commit(
 	cleaner->mark_space_used(
 	  i->get_paddr(),
 	  i->get_length(),
+	  ceph::encoded_sizeof_bounded<extent_info_t>(),
```

Comment:
This will still result in an underestimate of the total space used since the sum of extent_info_t's of a record won't be equal to the full size of the header (deltas, etc).  Is it worth adding this complexity just to reduce the severity of that underestimate?  Moreover, for usage ratio calculations, arguably a segment with a disproportionate amount of overhead really should be have a lower utilization ratio.  I'm not sure this is worth it.

Reply:
Agreed, dropped the commit.